airbyte-integrations/connectors/source-instagram/source_instagram/client/api.py

```diff
@@ -42,41 +63,60 @@ def __init__(self, api, *args, **kwargs):
     def list(self, fields: Sequence[str] = None) -> Iterator[dict]:
         """Iterate over entities"""
 
+    def filter_input_fields(self, fields: Sequence[str] = None):
+        return list(set(fields) - set(self.non_object_fields))
+
 
 class IncrementalStreamAPI(StreamAPI, ABC):
     @property
     @abstractmethod
     def state_pk(self):
         """Name of the field associated with the state"""
 
+    @property
+    @abstractmethod
+    def account_pk(self):
```

Comment:
This is just the cursor field, right? Can we call it that if so?

Reply:
Yes